picard/coverart/providers/__init__.py

```diff
@@ -17,23 +17,74 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+from picard import log, config
 from picard.plugin import ExtensionPoint
+from picard.ui.options import OptionsPage, register_options_page
 
 
 _cover_art_providers = ExtensionPoint()
 
 
+class ProviderOptions(OptionsPage):
+
+    """
+        Template class for provider's options
+    """
+
+    PARENT = "cover"
+    SORT_ORDER = 0
+    ACTIVE = True
+
+    options = []
```

Comment:
Removed in https://github.com/zas/picard/commit/f5821ca4aa0d24deb21b275974a103654438d0ca